test-utils/src/lib.rs

```diff
@@ -179,14 +179,14 @@ pub async fn wait_n_finalized_blocks(n: usize, url: &str) {
 	use substrate_rpc_client::{ws_client, ChainApi};
 
 	let mut built_blocks = std::collections::HashSet::new();
-	let mut interval = tokio::time::interval(Duration::from_secs(3));
+	let mut interval = tokio::time::interval(Duration::from_secs(2));
 	let rpc = ws_client(url).await.unwrap();
 
 	loop {
 		if let Ok(block) = ChainApi::<(), Hash, Header, ()>::finalized_head(&rpc).await {
 			built_blocks.insert(block);
 			if built_blocks.len() > n {
-				break
+				break;
```

Comment:
Is this semicolon critical? I see another place where it got added?

Reply:
Explicit break and return do require a semicolon as they're parsed as statements.